### PR TITLE
Add a roles page

### PR DESCRIPTION
**Pull request by svenfuchs**
This is a draft. Please feel free to edit and improve :)

### CODE REVIEW

File: roles.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+Rails Girls Summer of Code centers around teams. An ideal team has a pair of
+students, one or more coaches and a project expert, called mentor.
+
+##### Student
+
+Students obviously are the main characters in this program. They work on Open
+Source projects fulltime during summer, ideally in pairs. Everyone else on the
+program is doing their best to support them.
+
+Students also are the only ones who get paid by the program itself.
```
Comment (carpodaster) [inline roles.md]: "Getting paid" sounds like a 9-5 job to me, but I can't come up with a better term. "… who will receive monetary compensation"? Not sure.

File: roles.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+class: page page-roles
+current-sub: roles
+permalink: roles/
+current: roles
+---
+
+Rails Girls Summer of Code is run by a community of volunteers, and supported
+by companies.
+
+During the program people take on the following roles.
```
Comment (carpodaster) [inline roles.md]: (I like how this has the notion of a stage-play :smile: )

**Comment by sareg0** [comment]
Took a look and got the formatting ready for publish, and just made some minor edits so it would fit a nice format... If the text is to superfluous or silly, you can always change... It is a little entertaining ;)

**Comment by svenfuchs** [comment]
@sareg0 LOVE IT :heart:

Let's get this merged and out of the door!

**Pull request merged**